eZ/Publish/Core/Repository/Mapper/SortClauseMapper.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/**
+ * File containing SortClauseMapper class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
+ */
+namespace eZ\Publish\Core\Repository\Mapper;
+
+use eZ\Publish\API\Repository\Exceptions\NotImplementedException;
+use eZ\Publish\API\Repository\Values\Content\Location;
+use eZ\Publish\API\Repository\Values\Content\Query;
+use eZ\Publish\API\Repository\Values\Content\Query\SortClause;
+use eZ\Publish\Core\Base\Exceptions\InvalidArgumentException;
+
+class SortClauseMapper
```

Comment:
Even if it is internal, distinguishing the interface from the implementation gives us more flexibility, and makes BC easier if we need to make consequent changes to the SortClauseMapper's behaviour.

It doesn't cost much to make SortClauseMapper an interface, and rename the concrete implementation.